content/en/community/_index.html

```diff
@@ -122,7 +122,7 @@ <h2>Discussions</h2>
 
     <div id="twitter" class="community-resource">
       <a href="https://twitter.com/kubernetesio">
-        <img src="/images/community/twitter.png" alt="Twitter">
+        <img src="/images/community/x.png" alt="X.org">
       </a>
       <a href="https://twitter.com/kubernetesio">Twitter&nbsp;&#9654;</a>
```

Comment:
While we are updating the logo, let's also change the link text from Twitter to X for consistency.